src/index.ts

```diff
@@ -63,3 +65,9 @@ export {
     IVehicleLookupResponse,
     VehicleCondition,
 } from "./vehicles/types";
+export {
```

Comment:
It feels like we're not exporting enough types here - shouldn't we export the `*Request`-types here at least?

Reply:
We could, but it is not strictly required when we export the builder. But we probably should, I will change that.